plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -74,14 +75,19 @@ defaultCekCostModel = CostModel defaultCekMachineCosts defaultBuiltinCostModel
 defaultCostModelParams :: Maybe CostModelParams
 defaultCostModelParams = extractCostModelParams defaultCekCostModel
 
+defaultUnliftingMode :: UnliftingMode
+defaultUnliftingMode = UnliftingImmediate
+
```

Comment:
wait, i read in some comment above the 'deferred' is the default mode

Reply:
Are you sure? There's a comment about it being "preferred", but it does not make it default quite yet.

Reply:
we want deferred to be the default but immediate matches the current behaviour